fiboa_cli/datasets/hr.py

```diff
@@ -1,191 +1,169 @@
 from .commons.data import read_data_csv
-from .commons.admin import add_admin
-from ..convert_utils import convert as convert_
+from .commons.admin import AdminConverterMixin
+from ..convert_utils import BaseConverter
 import numpy as np
 
-SOURCES = "https://www.apprrr.hr/wp-content/uploads/nipp/land_parcels.gpkg"
+class Converter(AdminConverterMixin, BaseConverter):
+    sources = "https://www.apprrr.hr/wp-content/uploads/nipp/land_parcels.gpkg"
+    id = "hr"
+    short_name = "Croatia"
+    title = "Croatian Field Boundaries"
+    description = """
+    Field boundary data for Croatia, provided as part of national agricultural datasets.
 
-ID = "hr"
-SHORT_NAME = "Republic of Croatia"
-TITLE = "Croatian Agricultural Spatial Data"
-DESCRIPTION = """
-This dataset contains spatial data related to agricultural land use in Croatia, including ARKOD parcel information,
-environmentally sensitive areas, High Nature Value Grasslands, protective buffer strips around watercourses, and vineyard
-classifications. The data is crucial for managing agricultural activities, ensuring compliance with environmental regulations,
-and supporting sustainable land use practices.
-"""
+    This dataset contains spatial data related to agricultural land use in Croatia, including ARKOD parcel information,
+    environmentally sensitive areas, High Nature Value Grasslands, protective buffer strips around watercourses, and vineyard
+    classifications. The data is crucial for managing agricultural activities, ensuring compliance with environmental regulations,
+    and supporting sustainable land use practices.
+    """
 
-PROVIDERS = [
-    {
-        "name": "Agencija za plaćanja u poljoprivredi, ribarstvu i ruralnom razvoju",
-        "url": "https://www.apprrr.hr/prostorni-podaci-servisi/",
-        "roles": ["producer", "licensor"]
-    }
-]
-
-ATTRIBUTION = "copyright © 2024. Agencija za plaćanja u poljoprivredi, ribarstvu i ruralnom razvoju"
+    providers = [
+        {
+            "name": "Agencija za plaćanja u poljoprivredi, ribarstvu i ruralnom razvoju",
+            "url": "https://www.apprrr.hr/prostorni-podaci-servisi/",
+            "roles": ["producer", "licensor"]
+        }
+    ]
 
-LICENSE = {"title": "Prostorni podaci i servisi", "href": "https://www.apprrr.hr/prostorni-podaci-servisi/", "type": "text/html", "rel": "license"}
+    attribution = "copyright © 2024. Agencija za plaćanja u poljoprivredi, ribarstvu i ruralnom razvoju"
 
-COLUMNS = {
-    'id': 'id',
-    'land_use_id': 'crop_code',
-    'crop_name': 'crop_name',
-    'crop_name_en': 'crop_name_en',
-    'area': 'area',
-    'geometry': 'geometry',
+    license = {"title": "Prostorni podaci i servisi", "href": "https://www.apprrr.hr/prostorni-podaci-servisi/", "type": "text/html", "rel": "license"}
```

Comment:
```suggestion
    license = {
      "title": "Prostorni podaci i servisi",
      "href": "https://www.apprrr.hr/prostorni-podaci-servisi/",
      "type": "text/html",
      "rel": "license"
    }
```

For the sake of better readability